content/blog/2021/06/2021-06-17-libera-chat.adoc

```diff
@@ -0,0 +1,20 @@
+---
+:layout: post
+:title: "Jenkins IRC on Libera Chat"
+:tags:
+- community
+:author: markewaite
+opengraph:
+  image: /images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png
```

Comment:
Arrow could be moved to not overlap with text

Reply:
Fixed in a8188edb8cac2fd1d283e7ba8536adedbe1ee7e3 .  Thanks!